cypress/drafts/sms_gateway/Add_bulksms_gateway.feature

```diff
@@ -0,0 +1,22 @@
+Feature: BulkSMS gateway configurations can be added
+
+  Background:
+    Given the user navigated to the gateway configuration page
+
+  Scenario: The user successfully adds a BulkSMS gateway
+    When the user clicks on the add gateway button
+    And the user changes the gateway type to BulkSMS
+    And the user fills in the form data
+      | name | username | password |
+      | foo  | bar      | baz      |
+    And the user clicks on the add button
+    Then the user should be redirected to the gateway configuration page
+    And the newly added gateway should be listed in the table at the bottom
+
+  Scenario: The user wants to add a generic gate but doesn't supply the required data
```

Comment:
I'd use the complete name to be precise: Generic HTTP gateway

Reply:
Oh wait, from the following lines it seems this is actually about BulkSMS. In that case I'd modify this line to match that.